webapp/src/containers/Rhs/index.tsx

```diff
@@ -15,11 +16,25 @@ const Rhs = (): JSX.Element => {
     const {state, makeApiRequestWithCompletionStatus, getApiState} = usePluginApi();
     const connected = state.connectedReducer.connected;
 
+    const [totalLinkedChannels, setTotalLinkedChannels] = useState<ChannelLinkData[]>([]);
+    const [paginationQueryParams, setPaginationQueryParams] = useState<PaginationQueryParams>({
+        page: Constants.DefaultPage,
+        per_page: Constants.DefaultPageSize,
```

Comment:
```suggestion
        per_page: Constants.DefaultPerPage,
```
I think it will be better